kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -141,6 +142,7 @@ export default {
     theme,
     definitions,
     keenVars,
+    buttonAndLinkStyles,
```

Comment:
This is not being referenced here, are you exposing it purely for use in an external plugin?